src/javascript/app/common/redirect_banner.js

```diff
@@ -20,6 +23,20 @@ const RedirectBanner = (() => {
 
     const handleRedirect = () => {
         window.location.href = '/move-to-deriv/';
+    
+    };
+    
+    const handleRedirectROW = () => {
+        window.location.replace('https://binary.bot/move-to-deriv');
+        
+        // Check if param have ?binary-com-lp
+        const redirectBinary = new URLSearchParams(window.location.search);
+        if (redirectBinary.has('binary-com-lp') || Cookies.get('binary-com')){
+            // Set cookie if they wanted to stay at binary.com and no redirect
+            Cookies.set('binary-com', true, { expires: 15 });
+            window.location.href = 'https://binary.com';
```

Comment:
The logic should be like, you won't be redirecting, it will be in the same place. I f the logic doesn't match you redirect them. 

Also, U was checking, this 2 urls are also redirecting. They should not redirect.
Testlink 2: https://binary-static-git-fork-meenakshi-deriv-meenu-binary-com-883223.binary.sx/en/logged_inws.html?binary-bot-lp
Testlink 3: https://binary-static-git-fork-meenakshi-deriv-meenu-binary-com-883223.binary.sx/en/trading.html?binary-bot-lp